include/boost/gil/algorithm.hpp

```diff
@@ -734,10 +734,28 @@ void default_construct_pixels(View const& view)
 
 namespace detail {
 
+enum class copy_planarity_condition
+{
+    planar_2_planar,
+    interleaved_2_planar,
+    mixed_2_interleaved
+};
```

Comment:
Please, make `_2_`  read `_to_`. We ask for descriptive names, even of lengthy, and avoidance of abbreviations where possible.